src/ui/layout/Layout.vue

```diff
@@ -150,19 +152,19 @@
 </template>
 
 <script>
-import Inspector from '../inspector/Inspector.vue';
-import MctTree from './mct-tree.vue';
 import ObjectView from '../components/ObjectView.vue';
+import Inspector from '../inspector/Inspector.vue';
```

Comment:
Can we undo the import reordering? Unless there's some reason for it?

Reply:
my VScode is reordering I don't know why

Reply:
@vhenckel you could try disabling any linting extensions for the workspace, if you have any installed